src/lmic/radio_sx127x.c

```diff
@@ -1441,3 +1441,4 @@ void os_radio (u1_t mode) {
 ostime_t os_getRadioRxRampup (void) {
     return RX_RAMPUP_DEFAULT;
 }
+#endif
```

Comment:
add `// defined(CFG_sx1272_radio) || defined(...)` as a comment on the `#endif`. This is too many lines otherwise.

Reply:
Addressed in d80157b